server/src/main/java/org/elasticsearch/common/logging/TraceIdConverter.java

```diff
@@ -52,7 +52,7 @@ public static String getTraceId() {
     public void format(LogEvent event, StringBuilder toAppendTo) {
         String traceId = getTraceId();
         if (traceId != null) {
-            toAppendTo.append("\"trace.id\": \"" + traceId + "\"");
+            toAppendTo.append(traceId);
```

Comment:
What's the reason for this change? Is it so that we can reuse the traceId formatter in EcsJsonLayout.java? 

I wonder if this changes behaviour a bit, previously if traceId was null the format would not append anything, but now we might get "trace.id:null". Is that OK?

Reply:
Maybe notEmpty is smart enough ? :)

Reply:
yes, notEmpty will check if any of the variables in its "body" are evaluated